runtime/common/src/impls.rs

```diff
@@ -275,3 +278,160 @@ pub mod xcm {
 		}
 	}
 }
+
+#[cfg(test)]
```

Comment:
It's awesome to bring this in but it would be better to have the mock runtime in a `mock.rs` and allow for tests needing/using it to import it, just like we do for example for all the pallets individually.

Reply:
Ok, I agree with that 👍🏻